### ISSUE TITLE
[ERROR] vertex "ibm_scc_profile_attachment.scc_profile_attachment_instance"

### ISSUE THREAD

**Issue by lionelmace** [IBM-Cloud/terraform-provider-ibm#4820]
### Community Note
* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform IBM Provider Version

I cannot run terraform v1.58.0-beta0 anymore without facing this error `Your account is disabled` when calling the resource `ibm_scc_profile_attachment.scc_profile_attachment_instance`. My account is not disabled!

Full stack trace:
```
2023-09-25T14:48:44.329+0200 [ERROR] vertex "ibm_scc_profile_attachment.scc_profile_attachment_instance" error: CreateAttachmentWithContext failed Your account is disabled. Contact support.
{
    "StatusCode": 410,
    "Headers": {
        "Cf-Cache-Status": [
            "DYNAMIC"
        ],
        "Cf-Ray": [
            "80c36214fbf518a9-MRS"
        ],
        "Content-Length": [
            "130"
        ],
        "Content-Type": [
            "application/json; charset=utf-8"
        ],
        "Date": [
            "Mon, 25 Sep 2023 12:48:44 GMT"
        ],
        "Server": [
            "cloudflare"
        ],
        "Strict-Transport-Security": [
            "max-age=31536000; includeSubDomains"
        ],
        "X-Correlation-Id": [
            "8aab621d-fc2b-4703-b887-71741c43fe38"
        ],
        "X-Envoy-Upstream-Service-Time": [
            "19"
        ]
    },
    "Result": {
        "errors": [
            {
                "code": "Gone",
                "message": "Your account is disabled. Contact support."
            }
        ],
        "trace": "8aab621d-fc2b-4703-b887-71741c43fe38"
    },
    "RawResult": null
}
```

### Affected Resource(s)

* ibm_scc_profile_attachment.scc_profile_attachment_instance

### Steps to Reproduce

Here are the terraform scripts used
https://github.com/lionelmace/ibmcloud-utils/tree/master/terraform/scc

1. `terraform init`
1. `terraform apply`

**Comment by tyao117** [comment]
Based on your Terraform script , i am assuming that you are trying to use the same instance_id in different regions which can't be done. 

this line: is the base of my assumption. https://github.com/lionelmace/ibmcloud-utils/blob/ee1c688a7d4dc86e18dac83dc19a4838c6cdb272/terraform/scc/security-scc.tf#L53

**Comment by lionelmace** [comment]
@tyao117 No. I'm setting the instance_id created tin the same region `eu-de`.

**Comment by lionelmace** [comment]
The error `Your account is disabled `occurs when trying to target an SCC instance_id that does not exist anymore or has been deleted.
This error message should be replaced by something like this `SCC instance_id you're trying to target has been deleted. Please make sure you target the correct SCC instance_id.`
